packages/test/test-end-to-end-tests/src/test/gc/gcTreeSummaryHandles.spec.ts

```diff
@@ -179,8 +178,13 @@ async function submitAndAckSummary(
 /**
  * Validates whether or not a GC Tree Summary Handle should be written to the summary.
  */
-describeNoCompat("GC Tree stored as a handle in summaries", (getTestObjectProvider) => {
+describeNoCompat("GC Tree stored as a handle in summaries", (getTestObjectProvider, apis) => {
+	const {
+		containerRuntime: { ContainerRuntimeFactoryWithDefaultDataStore },
+	} = apis;
+
 	let provider: ITestObjectProvider;
+	// TODO:#4670: Make this compat-version-specific.
```

Comment:
I bet you can create a new class out of api.DataObjectFactory and this would work. I can probably take a crack at is as part of the issue you created.

Reply:
Yeah. I don't think it's super hard to fix--just wanted to pick off the low-hanging fruit in a first PR.